WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/Package Creation/ShippingLabelServicePackageList.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+import Yosemite
+
+struct ShippingLabelServicePackageList: View {
+    @Environment(\.presentationMode) var presentation
+    @StateObject var viewModel = ShippingLabelServicePackageListViewModel()
+    let packagesResponse: ShippingLabelPackagesResponse?
+    let safeAreaInsets: EdgeInsets
+
+    var body: some View {
+        LazyVStack(spacing: 0) {
+            ListHeaderView(text: Localization.servicePackageHeader, alignment: .left)
+                .padding(.horizontal, insets: safeAreaInsets)
+
+            /// Packages
+            ///
+            ForEach(viewModel.predefinedOptions, id: \.title) { option in
+
+                ListHeaderView(text: option.title.uppercased(), alignment: .left)
+                    .padding(.horizontal, insets: safeAreaInsets)
```

Comment:
Nit: I think there should be a divider underneath the list header?

![Simulator Screen Shot - iPhone 12 Pro - 2021-08-30 at 11 59 00](https://user-images.githubusercontent.com/5533851/131287501-f02d3df1-0124-4f8b-985b-db88814a36fa.png)

Reply:
This is based on the "Package Selected" screen, which also doesn't have a divider under the list headers:

![Simulator Screen Shot - iPhone 12 Pro - 2021-08-31 at 16 05 55](https://user-images.githubusercontent.com/8658164/131527741-2cb9e36b-d6fc-4d16-867a-ce5de116e54e.png)

That seems to be the design for most of the screens in Figma, but I do see a divider in the design for this screen in particular. @woocommerce/mobile-designers could you confirm whether there should be a divider under the header / at the top of the list on these screens? I can make any needed fixes for this in a followup PR.

Reply:
Hey @rachelmcr yes that's correct there should be a divider at the bottom of the header

Reply:
Thanks @adamzelinski I'll open a PR to fix that on the screens that are missing the divider.